client/src/lib/components/NetworkDropdown.svelte

```diff
@@ -18,7 +18,7 @@
       <Chevron />
     </div>
   </div>
-  <ul tabindex="-1" class="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-full text-white">
+  <ul tabindex="-1" class="dropdown-content menu p-2 shadow bg-base-100 rounded-box w-full text-white z-10">
```

Comment:
Good call with the `z-10`. I noticed that error but didn't have time to solve it.